### ISSUE TITLE
Txt format

### ISSUE THREAD

**Issue by kvasnyj** [zhongkaifu/Txt2Vec#1]
Hi!
Is it possible to use txt model in distance mode?

**Comment by zhongkaifu** [comment]
Yes. You can use following command to run distance mode by using txt model:

Txt2VecConsole.exe -mode distance -modelfile model.bin -maxword 70 -txtmodel 1
